observation/interferometric_pointing.py

```diff
@@ -22,10 +22,10 @@ class NoTargetsUpError(Exception):
 parser.add_option('-m', '--max-duration', type='float', default=None,
                   help='Maximum duration of the script in seconds, after which script will end '
                        'as soon as the current track finishes (no limit by default)')
-parser.add_option('--max-extent', type='float', default=1.0,
-                  help='Maximum extent in degrees, the script will scan ')
-parser.add_option('--number-of-steps', type='int', default=10,
```

Comment:
In the most recent commit I have changed the default to 5, which I am perfectly satisfied with at present.

On a side note I must say that disagree that "odd numbers" should be a preferred over "even numbers". Of course 1+3=4 is the minimum required for a reliable fit. However I expect that the benefit of improving the precision of individual fits by increasing the number of offset pointings will be rapidly out-weighed by the negative impact on the global pointing model fit of achieving fewer numbers of targets per hour.

Reply:
On odd versus even:

Since `num_off = opts.number_of_steps - 1`, it seems that one of the "steps" is the pointing at the boresight. I would prefer to have an even number of offsets around the target to avoid asymmetries like:
```
  o         o
       o  
        
       o
```
(`--number-of-steps=4`, or `num_off = 3`)

Reply:
i insist that it is at present simply a **_personal preference_** to insist on odd or even number of off-target points. it will remain that way until someone comes up with an analytic proof, or collects a rather extensive set of experimental evidence.

for the moment we both seem content to work with a default of 1+4. i do intend to generate a few different data sets to perform a gross sensitivity check. i will try to include 1+4 vs. 1+5.

Reply:
Or a personal hunch 😛 

As the number of steps grow large the difference will become very small anyway, so I'm only quibbling about 2, 3, 4, 5 and 6 steps.